src/templates/pycontw-2020/events/talk_list.html

```diff
@@ -30,19 +30,34 @@ <h3 id="talk-category-{{ category }}">{{ category }}</h3>
 
 	<ul class="talk-list">
 		{% for proposal in talk_list %}
-    <li>
+        <li>
 			<p>
 				{% url 'events_talk_detail' pk=proposal.pk as event_talk_detail_url %}
-      	{% blocktrans with proposal_title=proposal.title speaker_names=proposal|speaker_names_display trimmed %}
-      	<a class="talk-title" href="{{ event_talk_detail_url }}">{{ proposal_title }}</a> by {{ speaker_names }}
-      	{% endblocktrans %}
+				{% with proposal_title=proposal.title speaker_names=proposal|speaker_names_display %}
+				<a class="talk-title" href="{{ event_talk_detail_url }}">{{ proposal_title }}</a>
+				{% if proposal.labels %}
+				<span class="talk-label"><a href="#community-track">{% trans proposal.labels %}</a></span>
+				{% endif %}
+				{% blocktrans %} by {{ speaker_names }}{% endblocktrans %}
+				{% endwith %}
 			</p>
 		</li>
 		{% endfor %}
 	</ul>
 
+
 </article>
 
 {% endfor %}
 
+<section class="note">
+	<a name="community-track"></a>
+	<span class="talk-label">{% trans 'Community Track' %}</span>
+	<p>
+	{% blocktrans %}
+	This is the description of the community track
```

Comment:
@Lee-W I'll need your help on the description.

Reply:
社群軌是 PyCon TW 的一項新活動。我們選定數個熱門的主題，並且將同主題的精采演講打包成 N 場連續放送的形式，讓會眾能聽得過癮。我們捨棄了儘管專業但是死板的會議室，在有特色的咖啡廳、展演空間、甚至是古蹟內來舉辦上述各種主題的演講，讓會眾能更加舒適地且近距離地與他人互動。

The community track is a new event in PyCon TW. We choose the talks with specific topics and package them into the same session. We'll leave the conference rooms and go to characteristic cafes, exhibition spaces, and even monuments so that the attendees and speakers can interact with each other more comfortably and closely.